src/export-page/CourseExportPage.test.jsx

```diff
@@ -108,45 +95,45 @@ describe('<CourseExportPage />', () => {
+    const { getByText } = renderComponent();
     expect(getByText(stepperMessages.stepperPreparingDescription.defaultMessage)).toBeInTheDocument();
   });
   it('should show download path for relative path', async () => {
     axiosMock
       .onGet(getExportStatusApiUrl(courseId))
       .reply(200, { exportStatus: EXPORT_STAGES.SUCCESS, exportOutput: '/test-download-path.test' });
-    const { getByText, container } = render(<RootWrapper />);
+    const { getByText, container } = renderComponent();
     const startExportButton = container.querySelector('.btn-primary');
     fireEvent.click(startExportButton);
-    // eslint-disable-next-line no-promise-executor-return
-    await new Promise((r) => setTimeout(r, 3500));
-    const downloadButton = getByText(stepperMessages.downloadCourseButtonTitle.defaultMessage);
-    expect(downloadButton).toBeInTheDocument();
-    expect(downloadButton.getAttribute('href')).toEqual(`${getConfig().STUDIO_BASE_URL}/test-download-path.test`);
+    await waitFor(() => {
+      const downloadButton = getByText(stepperMessages.downloadCourseButtonTitle.defaultMessage);
+      expect(downloadButton).toBeInTheDocument();
+      expect(downloadButton.getAttribute('href')).toEqual(`${getConfig().STUDIO_BASE_URL}/test-download-path.test`);
+    }, { timeout: 4_000 });
   });
   it('should show download path for absolute path', async () => {
     axiosMock
       .onGet(getExportStatusApiUrl(courseId))
       .reply(200, { exportStatus: EXPORT_STAGES.SUCCESS, exportOutput: 'http://test-download-path.test' });
-    const { getByText, container } = render(<RootWrapper />);
+    const { getByText, container } = renderComponent();
     const startExportButton = container.querySelector('.btn-primary');
     fireEvent.click(startExportButton);
-    // eslint-disable-next-line no-promise-executor-return
-    await new Promise((r) => setTimeout(r, 3500));
-    const downloadButton = getByText(stepperMessages.downloadCourseButtonTitle.defaultMessage);
-    expect(downloadButton).toBeInTheDocument();
-    expect(downloadButton.getAttribute('href')).toEqual('http://test-download-path.test');
+    await waitFor(() => {
+      const downloadButton = getByText(stepperMessages.downloadCourseButtonTitle.defaultMessage);
+      expect(downloadButton).toBeInTheDocument();
+      expect(downloadButton.getAttribute('href')).toEqual('http://test-download-path.test');
+    }, { timeout: 4_000 });
   });
   it('displays an alert and sets status to DENIED when API responds with 403', async () => {
     axiosMock
       .onGet(getExportStatusApiUrl(courseId))
       .reply(403);
-    const { getByRole, container } = render(<RootWrapper />);
+    const { getByRole, container } = renderComponent();
     const startExportButton = container.querySelector('.btn-primary');
     fireEvent.click(startExportButton);
-    // eslint-disable-next-line no-promise-executor-return
-    await new Promise((r) => setTimeout(r, 3500));
```

Comment:
A bunch of tests were using hard-coded delays/sleeps. I replaced them with `waitFor` which is potentially faster, though in this case still ends up waiting ~3s compared to the hard-coded 3.5s.